trino-aws-proxy/src/main/java/io/trino/aws/proxy/plugins/filebasedcredentialsprovider/FileBasedCredentialsProviderPlugin.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.aws.proxy.plugins.filebasedcredentialsprovider;
+
+import com.google.inject.Module;
+import io.trino.aws.proxy.spi.TrinoAwsProxyServerPlugin;
+
+public class FileBasedCredentialsProviderPlugin
```

Comment:
This class is no longer needed right?

Reply:
The class `FileBasedCredentialsProviderPlugin` implements `TrinoAwsProxyServerPlugin` and this is how it is loaded. Another way is to directly provide a static binding for CredentialsProvider in `TrinoAwsProxyServerModule`. What do you suggest ?

Reply:
`FileBasedCredentialsProvider` is not a plugin. If users want it they would just install the Guice module. The Trino-AWS plugin semantics aren't needed.